t/unit/tasks/test_stamping.py

```diff
@@ -1227,81 +1221,3 @@ def test_chord_stamping_one_level(self, subtests):
 
         with subtests.test("sig_2_res has stamped_headers", stamped_headers=["stamp"]):
             assert sorted(sig_2_res._get_task_meta()["stamped_headers"]) == sorted(["stamp"])
-
-    @pytest.mark.usefixtures("depends_on_current_app")
```

Comment:
don't we need these features anymore?

Reply:
No - I implemented it based on a wrong understanding of what is `Task.replace()` thus adding a behavior that does not make sense.
So I'm removing some of the recently developed code I added (even though it works, it's not needed because its not correct that the replace will stamp the replaced task implicitly).